.github/workflows/software-environments-stable.yml

```diff
@@ -15,20 +15,13 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.8", "3.9", "3.10"]
-        runtime-version: ["0.0.3", "0.0.4"]
+        runtime-version: ["0.0.4", "0.1.0"]
         server: ["https://cloud.coiled.io", "https://staging.coiledhq.com"]
         include:
           - server: "https://cloud.coiled.io"
             token-name: COILED_BENCHMARK_BOT_TOKEN
           - server: "https://staging.coiledhq.com"
             token-name: COILED_STAGING_BENCHMARK_BOT_TOKEN
-          - python-version: "3.7"
-            runtime-version: "0.0.3"
-        exclude:
-          # `coiled-runtime=0.0.3` pins to a version of Dask
-          # that doesn't support Python 3.10
-          - python-version: "3.10"
-            runtime-version: "0.0.3"
```

Comment:
Huzzah

Reply:
🎉

Reply:
This should also mean no more flaky "Scheduler didn't phone home" failures